setup_wordpress.sh

```diff
@@ -1,3 +1,103 @@
 #!/bin/bash
+set -e
+echo '################## LEMP Stack Installation ####################'
+cd ~
+echo "------------------ Updating APT ------------------"
+echo
+sudo apt -y update && sudo apt -y upgrade
 
-echo 'This script should install and setup Wordpress'
+echo "------------------ Installing Nginx ------------------"
+sudo apt -y install nginx
+
+echo "------------------ Configuring Firewall ------------------"
+sudo ufw enable
+sudo ufw allow 'Nginx Full'
+
+echo "------------------ Installing MYSQL ------------------"
+sudo apt -y install mysql-server
+sudo mysql_secure_installation
+
+echo "------------------ Creating WordPress Database ------------------"
+read -p "Enter Your MYSQL Password: " mysql_password
+
+sudo mysql -u root -p$mysql_password << EOF
+CREATE DATABASE IF NOT EXISTS wordpress DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+CREATE USER 'wordpressuser'@'localhost' IDENTIFIED BY 'password';
+GRANT ALL PRIVILEGES ON wordpress.* to wordpressuser@'localhost';
+FLUSH PRIVILEGES;
+EOF
+
+echo "The Default Password Set For The 'WordPress' Database is 'password'. You Can Change It Later."
+
+echo "------------------ Installing PHP ------------------"
+sudo apt -y install php-fpm php-mysql
+
+echo "------------------ Creating SSL Certificate ------------------"
+sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/private/nginx-selfsigned.key -out /etc/ssl/certs/nginx-selfsigned.crt
+sudo openssl dhparam -out /etc/nginx/dhparam.pem 2048
+
+echo "------------------ Creating Configuration Snippet ------------------"
+cp /vagrant/self-signed.conf /etc/nginx/snippets/
+
+echo "------------------ Creating ssl-params.conf ------------------"
+cp /vagrant/ssl-params.conf /etc/nginx/snippets/
+
+echo "------------------ Installing PHP Extentions ------------------"
+sudo apt update
```

Comment:
what gives `apt update` here?

Reply:
I think it would be better to merge the php installations and then remove this part

Reply:
I mean you already have `apt update` at the beginning, what is the reason to use other?

Reply:
I think I need to remove it here.